gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/jupiter/core/v4/endpoint/loadbalancer/RandomLoadBalancer.java

```diff
@@ -13,29 +13,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.gravitee.gateway.jupiter.core.v4.endpoint.lb;
+package io.gravitee.gateway.jupiter.core.v4.endpoint.loadbalancer;
 
 import io.gravitee.gateway.jupiter.core.v4.endpoint.ManagedEndpoint;
 import java.util.List;
+import java.util.Random;
 
 /**
- * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author Guillaume LAMIRAND (guillaume.lamirand at graviteesource.com)
  * @author GraviteeSource Team
  */
-public class NoneLoadBalancerStrategy implements LoadBalancerStrategy {
+public class RandomLoadBalancer extends AbstractLoadBalancerStrategy {
 
-    private final List<ManagedEndpoint> endpoints;
+    private static final Random RANDOM = new Random(); //NOSONAR the random value is not used for security purposes here
```

Comment:
I don't understand the `NOSONAR` comment 🤔. `RANDOM` is used line 39

Reply:
Using Random make sonar complaining about security, asking you to use SecureRandom which is slower and not required here.